src/bundles/config.js

```diff
@@ -74,4 +87,16 @@ function getURLFromAddress (name, config) {
   }
 }
 
+const checkIfGatewayUrlIsAccessible = memoize(async (url) => {
+  try {
+    const { status } = await fetch(
+    `${url}/ipfs/bafybeiczsscdsbs7ffqz55asqdf3smv6klcw3gofszvwlyarci47bgf354`
```

Comment:
I figured that this request for empty directory fetches 80KB of HTML. 

There is _even better_ CID that is always present: [`bafkqaaa`](https://cid.ipfs.io/#bafkqaaa) (inlined empty multihash):

```suggestion
    `${url}/ipfs/bafkqaaa`
```